packages/gatsby-theme-newrelic/README.md

```diff
@@ -2855,6 +2887,68 @@ const MyComponent = () => {
 };
 ```
 
+### `useLocale`
+
+A hook that will get an object of information regarding the local language used within a component.
+
+```js
+import { useLocale } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Arguments**
+
+n/a
```

Comment:
Wondering if `n/a` means we're not sure or there are none? I see some Arguments also saying `None` or `none` or `There are no arguments for this hook so it would be good to keep this consistent

Reply:
Yes, i'll shore that up